rfcs/rdm-0080-bulk-importer.md

```diff
@@ -0,0 +1,104 @@
+Start Date: 2025-02-01
+- RFC PR: https://github.com/inveniosoftware/rfcs/pull/92
+- Authors: Aaron McCollough, Esteban G. Gabancho
+
+# Bulk importer
+
+## Summary
+
+Currently, InvenioRDM's only affordance for the bulk creation, import, and/or editing of records and files requires direct CLI-command-driven engagement with records and files APIs.
+
+GUI-based bulk importing and editing of records and files is a widely desired, highly useful feature, which will help to make the platform appealing to a much broader base of institutional users.
+
+The proposed feature is a beta version of a bulk importer for metadata (in CSV format) and associated files.
```

Comment:
are any other formats also planned?

Reply:
Initially, we are working only with CSV, but we are designing the tool so anyone can add their preferred format.